file/file.h

```diff
@@ -188,12 +214,17 @@ struct FileSystem {
     std::string file_name_;
   };
 
-  enum class RemoveDirectoryParameters { ThrowExceptionOnError, Silent };
-  static inline void RemoveDirectory(
-      const std::string& directory,
-      RemoveDirectoryParameters parameters = RemoveDirectoryParameters::ThrowExceptionOnError) {
-    if (::rmdir(directory.c_str())) {
-      if (parameters == RemoveDirectoryParameters::ThrowExceptionOnError) {
+  enum class RemoveDirParameters { ThrowExceptionOnError, Silent };
```

Comment:
Thanks! Taken care of in https://github.com/KnowSheet/Bricks/pull/62
ᐧ

Thanks,
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783


On Tue, Feb 10, 2015 at 11:58 PM, John Babak <notifications@github.com>
wrote:

> In file/file.h
> <https://github.com/KnowSheet/Bricks/pull/61#discussion_r24478625>:
>
> > @@ -188,12 +214,17 @@ struct FileSystem {
> >      std::string file_name_;
> >    };
> >
> > -  enum class RemoveDirectoryParameters { ThrowExceptionOnError, Silent };
> > -  static inline void RemoveDirectory(
> > -      const std::string& directory,
> > -      RemoveDirectoryParameters parameters = RemoveDirectoryParameters::ThrowExceptionOnError) {
> > -    if (::rmdir(directory.c_str())) {
> > -      if (parameters == RemoveDirectoryParameters::ThrowExceptionOnError) {
> > +  enum class RemoveDirParameters { ThrowExceptionOnError, Silent };
>
> MkDir but not RmDir?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/Bricks/pull/61/files#r24478625>.
>